src/pages/amman/pre-made-configs.md

```
@@ -16,7 +16,7 @@ const { LOCALHOST, tmpLedgerDir } = require("@metaplex-foundation/amman");
 module.exports = {
   validator: {
     killRunningValidators: true,
-    accountsCluster: "https://api.metaplex.solana.com",
+    accountsCluster: "https://api.devnet.solana.com	",
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 52

(MD010, no-hard-tabs)